src/mame/mr/nl_dribling.cpp

```diff
@@ -0,0 +1,494 @@
+#define TTL_74LS161_DIP TTL_74161_DIP
+#define TTL_74LS164_DIP TTL_74164_DIP
+
+
+// CONTRASTO HAS INVERTED LOGIC OR BUG?
+
+static NETLIST_START(LM339) 
+{
+// * CONNECTIONS:   NON-INVERTING INPUT
+// *                | INVERTING INPUT
+// *                | | POSITIVE POWER SUPPLY
+// *                | | | NEGATIVE POWER SUPPLY
+// *                | | | | OPEN COLLECTOR OUTPUT
+// *                | | | | |
+// .SUBCKT LM339    1 2 3 4 5
+	NET_MODEL("LM339A_TI_QIN PNP(IS=800.0E-18 BF=2.000E3)")
+	NET_MODEL("LM339A_TI_QMI NPN(IS=800.0E-18 BF=1002)")
+	NET_MODEL("LM339A_TI_QMO NPN(IS=800.0E-18 BF=1000 CJC=1E-15 TR=807.4E-9)")
+	NET_MODEL("LM339A_TI_QOC NPN(IS=800.0E-18 BF=20.29E3 CJC=1E-15 TF=942.6E-12 TR=543.8E-9)")
+	NET_MODEL("LM339A_TI_DX D(IS=800.0E-18)")
+	ALIAS(1, VI1.2)
+	ALIAS(2, VI2.2)
+	ALIAS(3, F10.ON)
+	ALIAS(4, Q3.E)
+	ALIAS(5, Q5.C)
+	DIODE(DP, "LM339A_TI_DX")
+	VCVS(E1, 1)
+	CCCS(F10, 1)
+	CS(IEE, 0.0001)
+	QBJT_EB(Q1, "LM339A_TI_QIN")
+	QBJT_EB(Q2, "LM339A_TI_QIN")
+	QBJT_EB(Q3, "LM339A_TI_QMO")
+	QBJT_EB(Q4, "LM339A_TI_QMI")
+	QBJT_EB(Q5, "LM339A_TI_QOC")
+	RES(RP, 46.300000)
+	VS(V1, 0)
+	VS(VI1, 0.75)
+	VS(VI2, 0.75)
+	NET_C(E1.OP, F10.IP)
+	NET_C(Q2.C, Q3.B, Q4.C, Q4.B)
+	NET_C(VI1.1, Q1.B)
+	NET_C(VI2.1, Q2.B)
+	NET_C(IEE.2, Q1.E, Q2.E)
+	NET_C(F10.IN, V1.1)
+	NET_C(F10.ON, IEE.1, DP.K, RP.1)
+	NET_C(F10.OP, Q1.C, Q3.C, E1.IP)
+	NET_C(V1.2, Q5.B)
+	NET_C(Q3.E, Q4.E, E1.ON, E1.IN, Q5.E, DP.A, RP.2)
+}
+
+// CALCIO_A and CALCIO_B differ only for the parameters of 3 capacitors.
+static NETLIST_START(calcio)
+{
+	DEFPARAM(CAP1, 33)
+	DEFPARAM(CAP2, 68)
+	ANALOG_INPUT(I_V5, 5)
+	ANALOG_INPUT(I_V6, 6)
+	ANALOG_INPUT(I_V12, 12)
+	QBJT_EB(Q8, "BC239C")
+	QBJT_EB(Q9, "BC239C")
+	ALIAS(INPUT, R45.2)
+	RES(R45, 420) 
+	RES(R46, RES_K(8.2))  
+	RES(R48, RES_K(220)) 
+	RES(R49, RES_K(56)) 
+	RES(R50, RES_K(56)) 
+	RES(R51, RES_M(1)) 
+	RES(R52, RES_K(68)) 
+	RES(R53, RES_K(82))  
+	RES(R54, RES_K(1.9))  // WAS POT
+	RES(R55, RES_K(10))
+	RES(R56, RES_K(100))
+	RES(R57, RES_K(10))
+	RES(R58, RES_K(3.9))
+	CAP(C29, CAP_U(4.2))
+	CAP(C30, CAP_N($(@.CAP2)))	
+	CAP(C31, CAP_N(150)) 
+	CAP(C32, CAP_U(4.7))
+	CAP(C33, CAP_N($(@.CAP1))) 
+	CAP(C34, CAP_N($(@.CAP1))) 
+	NET_C(INPUT, R46.1)
+	ALIAS(GND, R45.1)
+	NET_C(R46.2, C33.2, C34.1)
+	NET_C(C33.1, R49.1, C29.1)
+	NET_C(C34.2, R50.2, R51.1, Q8.B)
+	NET_C(R49.2, C30.2, R50.1)
+	NET_C(C30.1, GND)
+	NET_C(C29.2, R52.1, R51.2, R53.1, Q8.C)
+	NET_C(R52.2, I_V5)
+	NET_C(Q8.E, R54.2)
+	NET_C(R54.1, GND)
+	NET_C(R53.2, C31.1)
+	NET_C(C31.2, R48.1, R56.2, Q9.B)
+	NET_C(R48.2, I_V5)
+	NET_C(R56.1, GND)
+	NET_C(Q9.C, R55.1)
+	ALIAS(OUTPUT, Q9.C)
+	NET_C(R55.2, I_V5)
+	NET_C(Q9.E, R57.2, C32.2)
+	NET_C(R57.1, GND)
+	NET_C(C32.1, R58.2)
+	NET_C(R58.1, GND)
+}
+
+// STOP_PALLA AND PARATA are very similar. PARATA requires JFETs and 
+// MOSFET EMULATION doesn't work. So using the same circuit with different
+// values to emulate the sound.
+NETLIST_START(stop_palla_parata) 
+{
```

Comment:
This isn’t used directly outside the file – it should be `static`.

Reply:
If I declare them static I get this:
```
In file included from ../../../../../src/mame/mr/nl_dribling.cpp:3:
../../../../../src/lib/netlist/nl_setup.h:61:28: error: 'void netlist_LM339(netlist::nlparse_t&)' was declared 'extern' and later 'static' [-fpermissive]
   61 | #define NETLIST_NAME(name) netlist##_##name
```
even if I have removed them the NETLIST_EXTERNAL references.

Reply:
ah, found it. Fixed.